src/main/java/com/kodilla/ecommercee/service/CartService.java

```diff
@@ -35,6 +39,26 @@ public Cart createCart() {
         return cartRepository.save(new Cart());
     }
 
+    public Cart saveCart(Cart cart) {
+        return cartRepository.save(cart);
+    }
+
+    public Cart getCart(Long cartId) throws CartNotFoundException {
+        if(cartRepository.getOne(cartId) == null ){
+            throw new CartNotFoundException();
+        } else {
+        return cartRepository.getOne(cartId);
+        }
+    };
+
+    public void deleteCart(Long cartId) throws CartNotFoundException {
+        if(cartRepository.getOne(cartId) == null ){
```

Comment:
To samo co wyżej, lub akurat tu zrobić sobie w repository dodatkową metodę existById(Long id), która zwraca true, jeśli taki id istnieje